src/oidc-client.ts

```diff
@@ -236,6 +261,40 @@ export class OidcClient {
     return Promise.resolve(token);
   }
 
```

Comment:
We should document the introspectToken endpoint at the appropriate place in the readme (where the code example is).

Also I would add a note somewhere that manage and utilize refresh tokens now as well.